patches/MC-35514__fix_shipping_label_creation_issue__2.4.0.patch

```diff
@@ -0,0 +1,244 @@
+diff --git a/vendor/magento/module-shipping/Block/Adminhtml/Order/Packaging.php b/vendor/magento/module-shipping/Block/Adminhtml/Order/Packaging.php
```

Comment:
Please change `diff --git` to `diff -Nuar` as all patches have such format.

Reply:
Fixed.